rootfs/standard/var/www/mynode/api.py

```diff
@@ -313,4 +313,22 @@ def api_get_usb_info():
         data["status"] = "success"
     except Exception as e:
         data["data"] = str(e)
-    return generate_api_json_response(data)
\ No newline at end of file
+    return generate_api_json_response(data)
+
+@mynode_api.route("/api/get_lnbits_superuser")
+def api_get_lnbits_superuser():
+    check_logged_in()
+
+    data = {}
+    data["status"] = "error"
+    data["data"] = "UNKNOWN"
+    try:
+        info = ""
+
+        info += to_string(subprocess.check_output("cat /mnt/hdd/mynode/lnbits/.super_user", shell=True))
```

Comment:
Where does this file that stored the super_user info get created? I only see it getting read/deleted.

Reply:
Oh, it looks like it file is created by lnbits.